src/text.rs

```diff
@@ -756,13 +792,13 @@ fn layout(
     y: f32,
     context: &mut TextContextImpl,
     res: &mut TextMetrics,
-    paint: &Paint,
+    text: &TextSettings,
```

Comment:
text is I think a typical variable name of a string (like i is for loop iteration). I suggest to rename to text_settings.

Reply:
Yeah, I also had problems because of that in some functions and already named it text_settings there.